source/views/dictionary/list.js

```diff
@@ -77,21 +99,43 @@ export class DictionaryView extends React.Component {
     return <ListSeparator key={`${sectionId}-${rowId}`} />
   }
 
+  handleResults = (results: Object) => {
+    this.setState({results: results})
+  }
+
   render() {
+    if (!terms) {
+      return <LoadingView />
```

Comment:
This is an attempt to solve a timing issue. The data might not have been available at one point on the initial load. I am not so sure this is the best approach to solve it.